test/docker-compose.e2e-tests.yml

```diff
@@ -12,7 +12,6 @@ services:
       target: grapl-e2e-tests-build
     command: |
       bash -c "
-        wait-for-it grapl-provision:${WAIT_PORT} --timeout=250 &&
```

Comment:
It would be worthwhile to see if we can remove `wait-for-it` from the base layer of any of these containers now.

(We might not, given how they share so much... if we can, though, we should.)